packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -283,6 +283,7 @@ export class Eth2Gossipsub extends GossipSub {
     // See https://github.com/ChainSafe/lodestar/issues/5604
     setTimeout(() => {
       this.events.emit(NetworkEvent.pendingGossipsubMessage, {
+        emittedAt: Date.now(),
```

Comment:
milisecond precision may not be enough, BLS worker latency metrics are often < 1 ms, check the implementation there

Reply:
Updated.